spacy/util.py

```diff
@@ -675,6 +675,7 @@ def load_model_from_init_py(
     enable: Union[str, Iterable[str]] = _DEFAULT_EMPTY_PIPES,
     exclude: Union[str, Iterable[str]] = _DEFAULT_EMPTY_PIPES,
     config: Union[Dict[str, Any], Config] = SimpleFrozenDict(),
+    pipe_instances: Dict[str, Any] = SimpleFrozenDict(),
```

Comment:
Not having this broke `nlp = spacy.load("en_core_web_sm")`,  which wasn't caught by the test suite :|